src/main/resources/org/jenkinsci/plugins/ewm/restrictions/WhitelistUserRestriction/config.jelly

```diff
@@ -0,0 +1,6 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry title="${%Whitelisted users}">
+        <f:repeatableProperty field="usersList" add="${%Whitelist user}"/>
```

Comment:
Should the user list be made to look more like the user list [here](https://wiki.jenkins-ci.org/download/attachments/10059796/owndb.png?version=1&modificationDate=1202251054000)?